### ISSUE TITLE
Configure Fill-Opacity on Scatter Plot

### ISSUE THREAD

**Issue by rheubach** [unchartedelixir/uncharted#58]
It would be nice to allow graph creators the ability to adjust the Fill-opacity attribute on the circles placed on the scatter plot. You can find a set value for the attribute on the "scatter-dot" circle within the live_scatter.html file.

**Comment by rheubach** [comment]
@alcwhite would this be an easy configuration to add to the scatter plot?

**Comment by alcwhite** [comment]
@rheubach Yes. Would the same opacity apply to all circles, or should there be circle-specific configuration for this?

**Comment by markeasterling** [comment]
thanks Beau!
